$MirOS: ports/lang/python/2.3/patches/patch-setup_py,v 1.2 2005/11/15 19:16:58 tg Exp $
$OpenBSD: patch-setup_py,v 1.1.1.1 2003/12/31 17:38:33 sturm Exp $
--- setup.py.orig	Mon Jan 17 22:07:27 2005
+++ setup.py	Thu Jul 19 21:45:29 2007
@@ -647,7 +647,7 @@ class PyBuildExt(build_ext):
                 exts.append( Extension('resource', ['resource.c']) )
 
             # Sun yellow pages. Some systems have the functions in libc.
-            if platform not in ['cygwin', 'atheos']:
+            if platform not in ['cygwin', 'atheos'] and platform[0:6] != 'mirbsd':
                 if (self.compiler.find_library_file(lib_dirs, 'nsl')):
                     libs = ['nsl']
                 else:
@@ -1148,7 +1148,7 @@ def main():
           ext_modules=[Extension('struct', ['structmodule.c'])],
 
           # Scripts to install
-          scripts = ['Tools/scripts/pydoc', 'Tools/scripts/idle']
+          scripts = []
         )
 
 # --install-platlib
